RecoEcal/EgammaClusterProducers/python/correctedIslandEndcapSuperClusters_cfi.py

```diff
@@ -18,4 +18,11 @@
     recHitProducer = cms.InputTag("ecalRecHit","EcalRecHitsEE")
 )
 
+from Configuration.Eras.Modifier_pA_2016_cff import pA_2016
+from Configuration.Eras.Modifier_peripheralPbPb_cff import peripheralPbPb
+from Configuration.Eras.Modifier_pp_on_XeXe_2017_cff import pp_on_XeXe_2017
 
+from RecoHI.HiEgammaAlgos.HiCorrectedIslandEndcapSuperClusters_cfi import correctedIslandEndcapSuperClusters as _hiCorrectedIslandEndcapSuperClusters
+
+for e in [pA_2016, peripheralPbPb, pp_on_XeXe_2017]:
```

Comment:
It seems there should be an easy way to avoid this ever lengthening set of modifiers and just have one.. I'll try to suggest something

Reply:
I will make a new PR which adds a new era for 2017 pp ref run. I can implement your suggestion in the new PR.